tests/ssl_example_cert/cert.pem

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this a self-signed certificate? Are we sure we are not leaking anything here?